.github/workflows/build-publish.yaml

```diff
@@ -80,6 +89,8 @@ jobs:
           context: ./
           push: true
           file: ${{ matrix.package.path }}/Dockerfile.prod
+          build-args: |
+            ${{ matrix.package.identifier == 'boutique:frontend' && matrix.package.env_var && 'VITE_API_BASE_URL=https://api-boutique.jopacc.openpayments.directory' || '' }}
```

Comment:
We will have to specify the URL for the original Rafiki Boutique as well and also push them to different tags. Since this will build twice it will be overwrite the previous built image with `env_var: false` because both of them are pushed to the `latest` tag.

Reply:
I think they will be pushed in separate tags as the tag contains `matrix.package.name` in it, and in the matrix packages array, the items have different names: 

- name: rafiki-boutique-frontend
- name: rafiki-boutique-frontend-jopacc

And regarding the url for the original rafiki boutique, it was not set to anything before, and as such the default value is set from codebase: localhost:3004. 
You want me to add localhost:3004 as url in here as well, and not rely on the default from the codebase?

Reply:
Got confused by the multiple matrices.

You are right, but this means that we can get rid of the extra build here, since it's just testing the build process:

https://github.com/interledger/testnet/blob/784efeb08f6de6d03f8eaa06d6d385af5f768292/.github/workflows/build-publish.yaml#L13-L26

> You want me to add localhost:3004 as base_api_url for the original Rafiki Boutique as well, and not rely on the default from the codebase?

We will have to specify the actual API url, not `localhost:3004`, just like we do for the JoPACC one (in our current status it will have to be `https://api.rafiki.boutique`.)

Reply:
Oh so the artefacts of the build step that runs first are not used in the docker image build that is done as a step of the publish job? It's just so that it fails early?

So changes to the build job can be entirely reverted, the env variable added there makes no difference, all that is needed is the one done later, added as build-args to the docker build command?